R/verb-select.R

```diff
@@ -131,6 +131,28 @@ add_select <- function(lazy_query, vars) {
   )
 }
 
+select_can_be_inlined <- function(lazy_query, vars) {
+  vars_data <- op_vars(lazy_query)
+
+  bijective_or_not_distinct <- is_bijective_projection(vars, vars_data) ||
+      !is_true(lazy_query$distinct)
+
```

Comment:
I'd probably not name this (since the variable name is rather long and only used in one place) and instead inline in the logical expression below.